src/main/java/org/broadinstitute/consent/http/models/DataSet.java

```diff
@@ -168,4 +171,7 @@ public Integer getAlias() {
     public void setAlias(Integer alias) {
         this.alias = alias;
     }
+
+    public String getDatasetIdentifier() {return PREFIX + StringUtils.leftPad(alias.toString(), 6, "0"); }
```

Comment:
One way to make this more generic so other classes can use it is to move it to a public static method, then call that method from inside here to get what you want. For example, 
```
public String getDatasetIdentifier() {
    return parseAliasToIdentifier(this.getAlias()); 
}

public static String parseAliasToIdentifier(Integer alias) {
    return PREFIX + StringUtils.leftPad(alias.toString(), 6, "0"); 
}
```
That way, other classes that don't have a full `Dataset` object can use this method the same way `Dataset` does.